src/components/Workouts/Workouts.tsx

```diff
@@ -1,17 +1,19 @@
 import React, { FunctionComponent, useEffect, useState } from 'react';
-import Workout from '../../models/Workout';
-import Checkins from '../Checkins';
+import WorkoutViewModel from '../../view-models/WorkoutViewModel';
+import { makeWorkoutViewModel } from '../../view-models/WorkoutViewModelFactory';
```

Comment:
if you care to, you could export these from  `view-models/index.ts` which would make this import look like `import { makeWorkoutViewModel, WorkoutViewModel } from '../../view-models`